circleci.go

```diff
@@ -0,0 +1,344 @@
+package main
+
+import (
+	// "database/sql"
+	"fmt"
+	"net/http"
+	"encoding/json"
+	"io/ioutil"
+	"time"
+	"sync"
+
+	"github.com/pkg/errors"
+
+	"github.com/uptrace/bun"
+)
+
+const baseURL = "https://circleci.com/api/v1.1/project/github/determined-ai/determined"
+
+type Workflow string
+
+func (w *Workflow) UnmarshalJSON(b []byte) error {
```

Comment:
the idea here is to use the same model for the db and receiving the json?

Reply:
Yes.  This seemed less bad than having two separate structs with 80% duplication

Reply:
i think this is a bit error prone, but so is the other way. either way it's fine for this.